client/src/game/ui/HanabiCard.ts

```diff
@@ -858,25 +863,26 @@ export default class HanabiCard extends Konva.Group implements NodeWithTooltip {
   }
 
   // Update all UI pips to their state
-  updatePips(clueType: ClueType | null = null) {
+  updatePips() {
+    enum PipState {Hidden, Eliminated, Visible} // the order is important here
```

Comment:
the order shouldn't matter on an enum :( If you're hiding numbers on enums we can fix it

Reply:
I use Math.max on them. I will write numbers explicitly.